core/ui/PageTemplate.tid

```diff
@@ -1,4 +1,6 @@
 title: $:/core/ui/PageTemplate
+name: Core PageTemplate
```

Comment:
perhaps "default" instead of core? Let's see what @Jermolene thinks about it.

Reply:
@saqimtiaz - I've updated it to "Default PageTemplate" as the name and "the default tiddlywiki layout" as description